src/js/components/CheckBoxGroup/index.d.ts

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import { BoxProps } from '../Box';
+import { CheckBoxProps } from '../CheckBox' ;
+
+
+export interface CheckBoxGroupProps {
+  checked?: string | object;
```

Comment:
`checked?: string[] | number[];`?

Reply:
yes, `object` is definitely not the right type.

Reply:
Fixed.